impressum.html

```diff
@@ -0,0 +1,178 @@
+    <link rel="stylesheet" href="css/style.css">
+
+    
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.1/css/all.css"
+          integrity="sha384-O8whS3fhG2OnA5Kas0Y9l3cfpmYjapjI0E4theH4iuMD+pLhbf6JI0jIMfYcK3yZ" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+
+
+<body>
+ <!--
+      <nav class="navbar navbar-dark bg-dark>
+        <div class="container">
+          <a class="navbar-brand" href="index.html">YouVsVirus</a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#ftco-nav" aria-controls="ftco-nav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="oi oi-menu"></span> Menu
+          </button>
+
+          <div class="collapse navbar-collapse" id="ftco-nav">
+                <ul class="navbar-nav ml-auto style="white"">
+                 <li class="nav-item active"><a href="index.html" class="nav-link">Home</a></li>
+                <li class="nav-item active"><a href="index.html" class="nav-link">Impressum</a></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+ -->
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <a class="navbar-brand" href="#">YouVsVirus</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarText">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="index.html">Home </a>
+      </li>
+      <li class="nav-item  active">
+        <a class="nav-link" href="ímpressum.html">Impressum <span class="sr-only">(current)</span></a>
+      </li>
+    </ul>
+    <span class="navbar-text">
+    Stay home.
+    </span>
+  </div>
+</nav>
+
+
+    <!-- END nav -->
+    
+    <!-- Impressum
+
+================================================== -->
+
+  <div class="card" style="width: 800px;" >
+    <div class="card-body">
+     <h5 class="card-title"> <h1 class="display-4"> Impressum:</h1> </h5>
+    </div>
+  </div>
+
+  <div class="card" style="width: 800px;" >
+  <div class="card-body">
+    <h5 class="card-title"><p><span>Musterfirma/Musterunternehmer<br>Musterstraße 1<br>12345 Musterstadt<br>Deutschland</span></p>
+                           <p><span>Telefon: Telefonnummer<br>Telefax: Faxnummer<br>E-Mail:  <a href="mailto:youvsvirus@gmail.com">youvsvirus@gmail.com</a></span></p></h5>
```

Comment:
I like the Email. Is it real? If not we should register it.

We should change this line to "youvsvirs [at] gmail.com".
This will make it harder for spam crawlers to recognize it as an Email adress and bombard as with spam mails.

Reply:
ja
it exists

Reply:
done